.github/scripts/install.sh

```diff
@@ -0,0 +1,9 @@
+# Useful for debugging any issues with conda
+conda info -a
+
+conda create -q -n test-environment python=$PYTHON_VERSION numpy pyqt=4;
+source activate test-environment
+
+pip install --upgrade codecov
+pip install flake8
```

Comment:
There's now a `requirements.txt` and a `requirements_dev.txt` file. You may want to rebase your branch against latest `next`